salt/vault/secret_backends.sls

```diff
@@ -20,23 +20,23 @@ enable_mitx_aws_secret_backend:
 enable_pki_intermediate_backend:
   vault.secret_backend_enabled:
     - backend_type: pki
-    - mount_point: pki_int
```

Comment:
I think changing this to something like `pki-intermediate-ca` might be less confusing in the long run